docs/_static/bookshelf.css

```diff
@@ -1,6 +1,10 @@
+@import url('https://fonts.googleapis.com/css2?family=Noto+Color+Emoji&family=Nunito:ital,wght@0,200..1000;1,200..1000&family=Unbounded:wght@200..900&display=swap');
```

Comment:
We need to be aware when importing Google Font in term of privacy.
Is it not possible to directly embed the font?